deps/encoding/encoding.js

```diff
@@ -1008,7 +1009,14 @@ TextEncoder.prototype = {
       } while (last_byte !== EOF_byte);
       this._encoder = null;
     }
-    return new Buffer(bytes);
+    
+    var isModernBuffer = (
```

Comment:
This can be moved to the top level so that it's only executed once.

Reply:
Will do :-) Thanks for the review